### PR TITLE
Add option of shout for help

### PR DESCRIPTION
**Pull request by HimaniJV**

### CODE REVIEW

**Comment by HimaniJV** [comment]
Please check and merge

**Comment by HimaniJV** [comment]
Please review this, @casey-collab

File: english/explore-outside/journey-to-flaming-tower/journey-to-flaming-tower.md [hunk 1]
```diff
@@ -18,4 +18,6 @@ Do you:
 
 [Take your cellphone out, call Tank and ask for the nearest exit out of the Matrix.](exit/exit.md)
 
-[Think about how your towel could have been useful.](towel/towel.md)
\ No newline at end of file
+[Think about how your towel could have been useful.](towel/towel.md)
+
+[Shout for help](shout/shout.md)
```
Comment (casey-collab) [inline english/explore-outside/journey-to-flaming-tower/journey-to-flaming-tower.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/explore-outside/journey-to-flaming-tower/journey-to-flaming-tower.md (hunk 1 above)
Comment (cbuckey-uda) [inline english/explore-outside/journey-to-flaming-tower/journey-to-flaming-tower.md]: I do not see the `towel.md` file. Maybe you forgot to add and commit it?

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**